tests/test_glacier.py

```diff
@@ -0,0 +1,137 @@
+from common import BaseTest
+from c7n.executor import MainThreadExecutor
+from c7n.resources.glacier import Glacier
+
+
+class GlacierTagTest(BaseTest):
+
+    def test_glacier_tag(self):
+        session_factory = self.replay_flight_data('test_glacier_tag')
+        client = session_factory().client('glacier')
+        name = 'c7n-glacier-test'
+
+        client.create_vault(vaultName=name)
+        self.addCleanup(client.delete_vault, vaultName=name)
+
+        p = self.load_policy({
+            'name': 'glacier',
+            'resource': 'glacier',
+            'filters': [
+                {
+                    'type': 'value',
+                    'key': 'VaultName',
```

Comment:
this policy and assertion doesn't seem to be doing anything. afaics it can just be omitted.